jsapp/js/components/ccpmDocxReport201130.es6

```diff
@@ -242,13 +278,62 @@ const renderComment = (questionCode, questionName, parentState) => {
       const table = getTable2(tableData, 2, true, 20);
       if(table) dataToShow.push(table);
 }   );
+    const p12Result = getP12Question(parentState);
```

Comment:
Better to name the const and function by the question ID: P_IS02
The question number (P12) can change in the future.